### ISSUE TITLE
Can't import image inside tsx file in nextjs 9

### ISSUE THREAD

**Issue by michaelchiche** [twopluszero/next-images#21]
I am trying to use this plugin with nextjs 9.
When importing an image inside a .js file, it works fine, but fails when importing inside a tsx file.

**Comment by arefaslani** [comment]
Thank you for reporting. I'll fix it tomorrow.

**Comment by v1rtl** [comment]
I have the same issue on Next.js v9.0.2 and next-images v1.1.2

```tsx
// ...
import logo from './logo.svg'
// ...
<img src={logo} />
```

my ts config:

```ts
{
  "compilerOptions": {
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "jsx": "preserve",
    "lib": ["dom", "es2017"],
    "types": ["jest", "react", "styled-components"],
    "module": "esnext",
    "moduleResolution": "node",
    "noEmit": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "preserveConstEnums": true,
    "skipLibCheck": true,
    "sourceMap": true,
    "strict": true,
    "target": "es5",
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "resolveJsonModule": true,
    "isolatedModules": true
  },
  "exclude": ["node_modules"],
  "include": ["**/*.ts", "**/*.tsx", "**/*.svg"]
}

```

my `next.config.js`:

```js
const withImages = require('next-images')

module.exports = withImages({
  target: 'serverless'
})
```

**Comment by StefKors** [comment]
any progress on this issue?

**Comment by arefaslani** [comment]
Unfortunately not. ATM I'm too busy working on our startup. I checked it once but it had no progress because I hadn't enough time. Please contribute if you have any solution or wait until I can resolve this issue. Thank you.

**Comment by MilanLempera** [comment]
this is only typescript problem, you can avoid it by ts module definition for image types in d.ts file

e.g.
```
declare module "*.png" {
   const value: string;
   export = value;
}
```

**Comment by rtatarinov** [comment]
@MilanLempera Thanks. it works for me

**Comment by michaelchiche** [comment]
same for me! thanks

**Comment by arefaslani** [comment]
@MilanLempera Could you implement it in the library or in the readme file??

**Comment by MilanLempera** [comment]
@arefaslani https://github.com/twopluszero/next-images/pull/23

**Comment by arefaslani** [comment]
This issue is solved in version 1.2.0 thanks to @MilanLempera 🎉
